applications/osb-portal/src/service/WorkspaceService.tsx

```diff
@@ -27,9 +31,10 @@ class WorkspaceService {
   }
 
 
-  async fetchWorkspaces(): Promise<Workspace[]> {
+
+  async fetchWorkspaces(featured= false): Promise<Workspace[]> {
     // ToDo: pagination & size of pagination
-    const wspr: WorkspaceGetRequest = {};
+    const wspr: WorkspaceGetRequest = { q: 'publicable=' + (featured ? 'true' : 'False')};
```

Comment:
should this not be True (with capital)?

Reply:
Changed to lowecase (JSON compliant value)